smarts/env/gymnasium/hiway_env_v1.py

```diff
@@ -494,3 +504,12 @@ def smarts(self):
             smarts.core.smarts.SMARTS: The smarts simulator instance.
         """
         return self._smarts
+
+    @property
+    def seed(self):
+        """Returns the environment seed.
+
+        Returns:
+            int: Environment seed.
+        """
+        return current_seed()
```

Comment:
I am not sure if the fact that the seed is global for the current thread should be elevated to an issue.

So long as it ensures determinism for the space of a run it is currently enough.